File: src/main/java/team4/footwithme/team/api/request/TeamUpdateRequest.java
package team4.footwithme.team.api.request;

import team4.footwithme.team.service.request.TeamUpdateServiceRequest;

public record TeamUpdateRequest(String name, String description, String location) {
    public TeamUpdateServiceRequest toServiceRequest() {
        return new TeamUpdateServiceRequest(
                name, description, location
        );
    }
}

File: src/main/java/team4/footwithme/team/api/request/TeamUpdateRequest.kt
package team4.footwithme.team.api.request;

import jakarta.validation.constraints.Null;
import team4.footwithme.team.service.request.TeamDefaultServiceRequest;

public record TeamUpdateRequest(
    @Null
    String name,
    @Null
    String description,
    @Null
    String location
) {
    public TeamDefaultServiceRequest toServiceRequest() {
        return new TeamDefaultServiceRequest(
            name, description, location
        );
    }
}
